integration-tests/originate-known-contracts.ts

```diff
@@ -1,22 +1,28 @@
 import { CONFIGS } from './config';
 import { MichelsonMap, OriginateParams, RpcForger, TezosToolkit } from '@taquito/taquito';
-import { knownContract } from '../example/data/knownContract';
-import { knownBigMapContract } from '../example/data/knownBigMapContract';
 import { singleSaplingStateContractJProtocol } from './data/single_sapling_state_contract_jakarta_michelson';
 import { fa2ForTokenMetadataView } from './data/fa2-for-token-metadata-view';
 import { char2Bytes } from '@taquito/utils';
 import BigNumber from 'bignumber.js';
 import { codeViewsTopLevel } from './data/contract_views_top_level';
+import { knownBigMapContract } from './data/knownBigMapContract';
+import { knownContract } from './data/knownContract';
+const fs = require('fs');
```

Comment:
are we not able to use import for fs?

Reply:
Yes import would would work

Reply:
I think we should import for consistency

Reply:
Yes, thanks I made the change